<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuzzerBusinessActionWidget</class>
 <widget class="QWidget" name="BuzzerBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="fixedDurationCheckBox">
     <property name="text">
      <string>Fixed duration:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="durationWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="durationSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>86400</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationSuffixLabel">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
